<launch>
  <param name="target_ip" type="str" value="195.0.0.60" />
  <node pkg="external_interface" type="get_vehicle_input.py" name="get_vehicle_input" respawn="true" output="screen" />
</launch>